main.go

```diff
@@ -96,6 +96,13 @@ func main() {
 					return getBalance(ctx.Context, bs, tsk, actorAddr)
 				},
 			},
+			{
+				Name:        "fil-to-eth-address",
+				Description: "Converts an fX address to a 0x one if possible. Either pass a state CAR, tipset CIDs, or trust chain.love for a recent time",
+				Usage:       "<fX....>",
+				Flags:       append([]cli.Flag{}, stateFlags...),
```

Comment:
Feedback welcome on if there's a need for the lookback to be configurable (for this command or globally across more commands). My guess is probably not, but not sure.

Reply:
Resolved by #13 (which superseded #9)